packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -595,34 +642,17 @@ export default class DropdownCore extends React.Component<
                 ]}
             >
                 {this.maybeRenderSearchTextInput()}
-                {items.map((item, index) => {
-                    if (SeparatorItem.isClassOf(item.component)) {
-                        return item.component;
-                    } else {
-                        if (item.focusable) {
-                            focusCounter += 1;
-                        }
-                        const focusIndex = focusCounter - 1;
-                        return React.cloneElement(item.component, {
-                            ...item.populatedProps,
-                            key: index,
-                            ref:
-                                item.focusable &&
-                                this.state.itemRefs[focusIndex].ref,
-                            role: itemRole,
-                            onClick: () => {
-                                this.handleClickFocus(focusIndex);
-                                if (item.component.props.onClick) {
-                                    //$FlowFixMe
-                                    item.component.props.onClick();
-                                }
-                                if (item.populatedProps.onClick) {
-                                    item.populatedProps.onClick();
-                                }
-                            },
-                        });
-                    }
-                })}
+
+                {itemsList.length < 100 ? (
```

Comment:
I feel like this should be a constant at the top of this file (and potentially something that should be configurable - but we can explore that later).